config/testgrids/kubernetes/sig-cli/config.yaml

```diff
@@ -19,3 +19,8 @@ dashboards:
       description: unit, integration and cmd tests for master branch
 
 - name: sig-cli-misc
+  dashboard_tab:
```

Comment:
This section might not be necessary given the annotations in the YAML in the jobs directory (not sure)

Reply:
Yes, you'll need to delete this. This dashboard tab is defined twice (once here, and once as a Prow Annotation).

Keep the dashboard. `- name: sig-cli-misc`